deploy/addons/aliyun_mirror.json

```
@@ -24,7 +24,7 @@
   "k8s.gcr.io/sig-storage/csi-snapshotter": "registry.cn-hangzhou.aliyuncs.com/google_containers/csi-snapshotter",
   "k8s.gcr.io/sig-storage/csi-provisioner": "registry.cn-hangzhou.aliyuncs.com/google_containers/csi-provisioner",
   "registry": "registry.cn-hangzhou.aliyuncs.com/google_containers/registry",
-  "quay.io/nixpanic/glusterfs-server": "registry.cn-hangzhou.aliyuncs.com/google_containers/glusterfs-server",
+  "docker.io/gluster/gluster-centos": "registry.cn-hangzhou.aliyuncs.com/google_containers/glusterfs-server",
```

```suggestion
  "gluster/gluster-centos": "registry.cn-hangzhou.aliyuncs.com/google_containers/glusterfs-server",
```
If it's from DockerHub we omit `docker.io`, it will work without, but for consistency sake could you mind removing it, thanks!

---

it would be better to use docker.io (fully qualified name), but to keep this PR small and reverent, how about a follow up PR to change all of them to docker.io ?

---

Works for me

---

https://github.com/kubernetes/minikube/issues/14520